.travis.yml

```diff
@@ -9,6 +9,9 @@ services:
 env:
   - TOX_ENV=py27
 
+git:
+  submodules: false
+
```

Comment:
Because your modules are submodules of submodules? Just asking because the extra role just works fine.